sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -166,6 +168,34 @@ def _get_spot_pricing_table(region: str) -> pd.DataFrame:
     return df
 
 
+def _patch_p4de(region: str, df: pd.DataFrame,
+                pricing_df: pd.DataFrame) -> pd.DataFrame:
+    # Hardcoded patch for p4de.24xlarge, as our credentials doesn't have access
+    # to the instance type.
+    # Columns:
+    # InstanceType,AcceleratorName,AcceleratorCount,vCPUs,MemoryGiB,GpuInfo,
+    # Price,SpotPrice,Region,AvailabilityZone
+    for zone in df[df['Region'] == region]['AvailabilityZone'].unique():
+        df = df.append(pd.Series({
+            'InstanceType': 'p4de.24xlarge',
+            'AcceleratorName': 'A100-80GB',
+            'AcceleratorCount': 8,
+            'vCPUs': 96,
+            'MemoryGiB': 1152,
+            'GpuInfo':
+                ('{\'Gpus\': [{\'Name\': \'A100-80GB\', \'Manufacturer\': '
+                 '\'NVIDIA\', \'Count\': 8, \'MemoryInfo\': {\'SizeInMiB\': '
+                 '40960}}], \'TotalGpuMemoryInMiB\': 327680}'),
```

Comment:
Should these memory be ~80GB?

Do we have documentation somewhere on the diff b/t SizeInMiB and TotalGpuMemoryInMiB?

Reply:
Ahh good catch! Fixed. Thanks!

The `SizeInMiB` seems to be the memory of a single GPU and `TotalGpuMemoryInMiB` is the total memory for 8 GPUs